core-libs/setup/ssr/error-handling/server-error-response-handlers/server-error-response-handlers.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ErrorRequestHandler } from 'express';
+import {
+  CmsPageNotFoundServerErrorResponse,
+  UnknownServerErrorResponse,
+} from '../server-error-response/cx-server-error-response';
+
+/**
+ * Handles CmsPageNotFoundServerErrorResponse which results in a fallback to client side rendering.
+ * The document content is sent to the client with the appropriate 404 status code.
+ *
+ * @param documentContent The document content to be sent to the client.
+ * @returns The error request handler.
+ */
+export const handleCmsPageNotFoundErrorResponse =
+  (documentContent: string): ErrorRequestHandler =>
```

Comment:
nitpick: The name or the JSDoc of the function is a misleading. The function's name and JSDocs tell this function handles the server error response. But it's rather a getter function that returns a handler. 😅 
I'm having hard time finding a better name. `handleCmsPageNotFoundErrorResponse` vs `getCmsPageNotFoundErrorResponseHandler` (I'm not convinced 😕  this name is too long)
But at least we can improve the JSDoc